spec/board_spec.rb

```diff
@@ -135,6 +135,34 @@
       expect(board.valid_placement?(submarine, ["C2", "D3"])).to be false
       expect(board.valid_placement?(cruiser, ["A1", "B2", "C3"])).to be false
     end
+  end
+
+  describe '#place' do
+    it 'places ship in cells if placement is valid' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
```

Comment:
Putting note here to come back to the edge case of when user tries to place ship on invalid placement. What would be the return? How will it inform the future turn methods?